src/geode/basic/progress_logger.cpp

```diff
@@ -33,18 +33,18 @@
 
 namespace
 {
-    static constexpr absl::Duration SLEEP = absl::Seconds( 1 );
-}
+    constexpr absl::Duration SLEEP = absl::Seconds( 1 );
```

Comment:
warning: invalid case style for constexpr variable 'SLEEP' [readability-identifier-naming]

```suggestion
    constexpr absl::Duration sleep = absl::Seconds( 1 );
```